### DISCUSSION TITLE
TypeError: Cannot read property 'getPrefixCls' of undefined

### DISCUSSION THREAD

**Discussion by saikrishnabanda5** [ant-design/ant-design#36089] [Q&A]
I am getting error as TypeError: Cannot read property 'getPrefixCls' of undefined when i use Progress in antd .
Please help me

import { Progress } from 'antd';

true ? positionData / 26 : 0}
showInfo={false}
type="line"
strokeColor="black"
strokeWidth="2px"
trailColor="#b6b6b6"
// prefixCls="rc-tabs"
/>

I have used next js . During the build time i am getting this error

![172542247-a94a9d7d-7c0d-48a3-8f2c-4f634449260c](https://user-images.githubusercontent.com/54619513/174094947-fd45e36d-7eb7-4a6d-9fc8-4ec259964e2b.png)

**Comment by yoyo837** [comment]
Please open an issue.

**Comment by thanggggg1** [comment]
does it have solution

**Comment by pierregober** [comment]
Following... I have this same issue when using a notification

**Reply by viktorelmer** [reply]
It's the same for me, every time I try to send a notification, I get this error.
But I'm sure everything was fine yesterday.

**Comment by fesaza** [comment]
Have the same issue when I use the Select component

**Comment by mattmanik** [comment]
I faced the same issue. Solved it by adding {contextHolder} to my component's return snippet.

**Reply by Khadija-Hussain** [reply]
its not working for me , giving this error while using select

**Reply by yoyo837** [reply]
Please provide a reproduction and feel free to open a new issue for it.

**Reply by talantiarun** [reply]
yes it solved my issue. Adding {contextHolder} in jsx solved the issue.
